src/appTheme.ts

```diff
@@ -196,5 +196,9 @@ export const appTheme: ThemeConfig = {
       headerBg: 'var(--j2-color-bg-layout)',
       rowHoverBg: 'var(--j2-color-bg-layout)',
     },
+    Tag: {
+      defaultBg: 'var(--j2-color-fill-secondary)',
+      green: 'var(--j2-color-success-bg)',
```

Comment:
We'll have to add more colors here, but this seems like a good way to sync with design on what these mean

Reply:
Nevermind, apparently setting the `defaultBg` fixed my issue! This isn't necessary right now at least